source/connect/cli-options.txt

```diff
@@ -0,0 +1,52 @@
+.. _cli-options:
+
+----------------------------
+Compass Command Line Options
+----------------------------
+
+The following page provides an overview on how set command line options for 
```

Comment:
[missing word]

I think we're missing "to" on line 7.

Reply:
Thank you and done!